### PR TITLE
feature/v3.x.x/1069: Displaying "Stalled..." when effective bytes per second is 0

### PR DESCRIPTION
**Pull request by Dzejkop**
### Checklist
1. [x] Changelog entry
2. [x] Review requested
3. [ ] Review approved
4. [x] `Version.cs` update - if required

### CODE REVIEW

**Comment by Dzejkop** [comment]
Don't merge yet - I discovered an error

**Comment by Dzejkop** [comment]
Ok to merge

File: Assets/PatchKit Patcher/Scripts/Version.cs [hunk 1]
```diff
@@ -3,8 +3,8 @@ namespace PatchKit.Unity.Patcher
     public static class Version
     {
         public const int Major = 3;
-        public const int Minor = 10;
```
Comment (witcher112) [inline Assets/PatchKit Patcher/Scripts/Version.cs]: This number is updated right before creating release candidate, no need to do it right now.

File: Assets/PatchKit Patcher/Scripts/AppUpdater/Commands/DownloadPackageCommand.cs [hunk 2]
```diff
@@ -73,7 +74,11 @@ public override void Execute(CancellationToken cancellationToken)
 
             downloader.DownloadProgressChanged += bytes => { _status.Bytes.Value = bytes; };
 
-            downloader.Download(cancellationToken);
+            using (_status.BytesPerSecond.Subscribe(bps =>
+                _status.Description.Value = bps > 0 ? "Downloading package..." : "Stalled..."))
```
Comment (witcher112) [inline Assets/PatchKit Patcher/Scripts/AppUpdater/Commands/DownloadPackageCommand.cs]: I would use a small threshold while checking `bps` (like `0.01`). It's a `double` so we might encounter some floating point precision issues.

**Pull request merged**